types/react-tagsinput/index.d.ts

```diff
@@ -32,6 +32,7 @@ declare namespace TagsInput {
         readonly getTagDisplayValue: (tag: Tag) => string;
         readonly onRemove: (tagIndex: number) => void;
         readonly tag: Tag;
+        readonly key: number;
```

Comment:
```suggestion
        readonly key: string | number;
```
A key can be a string or number.